libaleth-interpreter/VM.h

```diff
@@ -119,14 +118,14 @@ class VM
 
     const evmc_tx_context& getTxContext();
 
-    void throwOutOfGas();
-    void throwInvalidInstruction();
-    void throwBadInstruction();
-    void throwBadJumpDestination();
-    void throwBadStack(int _removed, int _added);
-    void throwRevertInstruction(owning_bytes_ref&& _output);
-    void throwDisallowedStateChange();
-    void throwBufferOverrun(intx::uint512 const& _enfOfAccess);
+    static void throwOutOfGas();
```

Comment:
Maybe better to make them inline or/and free functions

Reply:
Or maybe there's not much sense in these helpers at all, and we can just put `throw`s inline

Reply:
This was done previously for performance reasons. I plan to rework this in separate PR with benchmarks.

Reply:
I did inlining. From quick benchmarks, the inlining commit gives ~3% improvement. In total the PR gives up to 13% improvement.